test/fluree/db/ledger/general/invoice_tests.clj

```diff
@@ -97,7 +97,74 @@
+                          :_role/rules "_rule$viewOrgEmployees"}
+                         ;; Add rule for query of _user(s) that checks org(s) they belong to and verifies same as requesting user
+                         {:_id                     "_rule$viewOrgEmployees",
+                          :_rule/id                "viewOrgEmployees",
+                          :_rule/doc               "Allow employees to see others in same org.",
+                          :_rule/collection        "_user",
+                          :_rule/fns               ["_fn$userSameOrg"],
+                          :_rule/ops               ["query"],
+                          :_rule/collectionDefault true}
+                         ;; add fn that checks user's 'orgs' context value contains the org the respective _user belongs to
+                         {:_id      "_fn$userSameOrg",
+                          :_fn/name "userSameOrg",
+                          :_fn/code "(contains? (ctx \"orgs\") (get (?sid) \"org/_employees\"))",
+                          :_fn/doc  "Returns truthy if _user is in same org(s) as requesting identity."}
+                         ;; add an extra user in the same org as Scott, so we can see if Scott can see them
+                         {:_id      "_user$fannie"
+                          :username "fannie"}
+                         {:_id       ["org/name" "NewCo"]
+                          :employees ["_user$fannie"]}]
+          tx-resp       (async/<!! (fdb/transact-async (basic/get-conn) test/ledger-invoice txn))
+          scott-db      (basic/get-db test/ledger-invoice {:syncTo (:block tx-resp)
+                                                           :auth   ["_auth/id" (:auth scott)]})
+          antonio-db    (basic/get-db test/ledger-invoice {:syncTo (:block tx-resp)
+                                                           :auth   ["_auth/id" (:auth antonio)]})
+          root-db       (basic/get-db test/ledger-invoice {:syncTo (:block tx-resp)})
+          query         {:select ["_user/username"]
+                         :from   "_user"}
+          root-q-res    @(fdb/query root-db query)
+          scott-q-res   @(fdb/query scott-db query)
+          antonio-q-res @(fdb/query antonio-db query)]
+      ;; root db sees all users:
+      (is (= root-q-res
+             [{"_user/username" "fannie", :_id 87960930223084}
+              {"_user/username" "antonio", :_id 87960930223083}
+              {"_user/username" "scott", :_id 87960930223082}
+              {"_user/username" "freddie", :_id 87960930223081}]))
+      ;; scott can see only _user in the same org
+      (is (= scott-q-res
+             [{:_id 87960930223084, "_user/username" "fannie"}
+              {:_id 87960930223082, "_user/username" "scott"}]))
+      ;; antonio can only see himself, no other users are in same org
+      (is (= antonio-q-res
+             [{"_user/username" "antonio", :_id 87960930223083}])))))
+
```

Comment:
Love the addition of tests!

One thing to consider is that when we migrate this test ns to be more idiomatic, each `deftest` will need to be runnable in isolation w/ its own randomly-named ledger. In here the second `deftest` relies on running after the first one & in the same ledger. It's fine if you want to leave changing that until we get to migrating this ns. Or you could change that now or even fully migrate the test ns to idiomatic if you want to / have time (I [documented the process here](https://github.com/fluree/ledger/blob/main/test/CONVERT_TO_IDIOMATIC.md)).

I think whichever option you choose is fine, just wanted to point it out.